src/compiler/python_generator.cc

```diff
@@ -236,6 +236,10 @@ bool PrintBetaServicer(const ServiceDescriptor* service,
              service->name());
   {
     IndentScope raii_class_indent(out);
+    out->Print("\"\"\"The Beta API is deprecated for 0.15.0 and later.\n"
+    "\nIt is recommended to use the GA API for all further purposes. This\n"
```

Comment:
Here and below, after "the GA API", please add a parenthetical phrase explaining what the GA API is to the user who may not have heard the term before. Something like `(the classes and functions in this file without "beta" in their names)`? Any thoughts on wordsmithing that?